github-webhook.js

```diff
@@ -227,10 +231,18 @@ function handleRules (logStream, rules, event) {
       }
     })
 
+    cp.stdout.pipe(past)
+    prefixStream(cp.stderr, '! ').pipe(past)
+
     if (logStream) {
-      prefixStream(cp.stdout, 'stdout: ').pipe(logStream, { end: false })
-      prefixStream(cp.stderr, 'stderr: ').pipe(logStream, { end: false })
+      past.pipe(logStream, {end: false})
```

Comment:
spacing, run this through the linter, hopefully it'll tell you to add spaces either side of the `{}`